CTestCustom.cmake

```diff
@@ -31,30 +94,33 @@
 #     )
 # endif()
 
-# examples from lapack
-# SET(CTEST_CUSTOM_WARNING_EXCEPTION
-#   ${CTEST_CUSTOM_WARNING_EXCEPTION}
+#------------------------------------------------------------------------------#
+# EXAMPLES FROM LAPACK
 
-#   # Common warning when linking ATLAS built with GNU Fortran 4.1 and building
-#   # with GNU Fortran 4.4.  It can be safely ignored.
-#   "libgfortran.*may conflict with libgfortran"
+# SET(CTEST_CUSTOM_WARNING_EXCEPTION ${CTEST_CUSTOM_WARNING_EXCEPTION}
```

Comment:
This may well not matter since it's commented, but I think the right paren is missing